testing/jsm/helpers.js

```diff
@@ -274,8 +265,8 @@ function inOrder(eventsArray, eventsSpecs) {
 
 function getEventTime(event) {
 
-    // the dvce_created_tstamp
-    return event["event"]["parameters"]["dtm"];
+    // the derived_tstamp
+    return event["event"]["derived_tstamp"];
```

Comment:
Did you intend to switch from the device created timestamp to the derived timestamp here? These are two different timestamps. I think `derived_tstamp` might be a better choice than device timestamp but just pointing it out. `collector_tstamp` might be an even better choice as it is more predictable than `derived_tstamp` but not essential for this PR.

Reply:
Yes, this was indeed an intended switch, now that the `derived_tstamp` can be accessed in Micro's response. For switching also took into account:
 - https://snowplowanalytics.com/blog/2015/09/15/improving-snowplows-understanding-of-time/
 - https://discourse.snowplowanalytics.com/t/which-timestamp-is-the-best-to-see-when-an-event-occurred/538

Reply:
Thats cool, `derived_tstamp` is fine by me.